src/quo/components/avatars/collection_avatar/style.cljs

```diff
@@ -2,10 +2,17 @@
   (:require
     [quo.foundations.colors :as colors]))
 
+(defn- get-dimensions
+  [size]
+  (case size
+    :size-24 24
+    :size-20 20
+    nil))
```

Comment:
Why is the size given as a prefixed keyword `:size-`?

Reply:
having lots of random integers used across the application is not the cleanest pattern and can easily lead to changes that can break many things unknowingly. The team came to a decision a while back to use this as a standard approach for passing size props to components. See: https://github.com/status-im/status-mobile/pull/17279